src/status_im/contexts/keycard/create/events.cljs

```diff
@@ -83,7 +83,7 @@
  (fn [{:keys [db]} [error]]
    (if (or (= error :keycard/error.keycard-empty)
            (and (get-in db [:keycard :application-info :initialized?])
-                (= error :keycard/error.keycard-wrong-profile)))
+                (= error :keycard/error.keycard-not-empty)))
```

Comment:
We can completely eliminate this check if we simply pass the key-uid while creating the account. Do you know how to obtain the key-uid from the seed phrase?

Reply:
cc @flexsurfer @ilmotta

Reply:
we need to move account generation here then, but it might make logic more complex

Reply:
im reworking create flow, so i'll take a look